lib/message_request.js

```diff
@@ -77,6 +78,18 @@ Request.buildUnsubscribe = function (scope) {
   return new Request(message);
 };
 
+Request.parseMessageFromData = function (data) {
```

Comment:
I'm -1 to including a class method wrapping JSON.parse. I would prefer something with a signature like  `Request.createFromString(String) => RequestMessage` that included JSON.parse and instantiated a Request object, which it returned.